src/incubator/expandableOverlay/index.tsx

```diff
@@ -12,6 +12,12 @@ export interface RenderCustomOverlayProps {
   toggleExpandable: () => void;
 }
 
+export interface ExpandableOverlayMethods {
```

Comment:
You can extend this interface in the one above (RenderCustomOverlayProps), just for making it look better

Reply:
Done